src/test/java/org/eolang/speco/SpecoTest.java

```diff
@@ -0,0 +1,165 @@
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.eolang.speco;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Arrays;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.SystemUtils;
+import org.eolang.jucs.ClasspathSource;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.io.CleanupMode;
+import org.junit.jupiter.api.io.TempDir;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.yaml.snakeyaml.Yaml;
+
+/**
+ * Packs tests.
+ *
+ * @since 0.0.1
+ */
+class SpecoTest {
+
+    @Tag("fast")
+    @ParameterizedTest
+    @ValueSource(strings = {"simple"})
+    public void convertsFromXmir(
+        final String title,
+        @TempDir(cleanup = CleanupMode.NEVER) final Path temp) throws IOException {
+        final Path base = Path.of(
+            "src", "test", "resources",
+            "org", "eolang", "speco",
+            "xmir", title
+        );
+        new Speco(base.resolve("in"), temp, false).exec();
+        final Path reference = base.resolve("out");
+        for (final Path path : Files.newDirectoryStream(reference)) {
+            MatcherAssert.assertThat(
+                String.format(
+                    "Files %s in %s and %s are different",
+                    path.getFileName(),
+                    temp,
+                    reference
+                ),
+                Files.readAllLines(temp.resolve(path.getFileName())),
+                Matchers.equalTo(
+                    Files.readAllLines(path)
+                )
+            );
+        }
+    }
+
+    @Disabled
+    @Tag("fast")
+    @ParameterizedTest
+    @ClasspathSource(value = "org/eolang/speco/packs", glob = "**.yaml")
+    public void convertsFromEo(
+        final String pack,
+        @TempDir(cleanup = CleanupMode.NEVER) final Path temp) throws IOException {
```

Comment:
@advasileva Do you need `cleanup = CleanupMode.NEVER`? It seems that it will leave garbage after each test run

Reply:
@volodya-lombrozo This policy [fixes](https://github.com/objectionary/speco/pull/32/commits/95d2f9a29f84df2fd4ea35117b9b32be3c711a14) tests failures on windows with Java 11, [e.g.](https://github.com/objectionary/speco/actions/runs/4097463770/jobs/7065856235)

Reply:
I tried several ways to fix it, but it didn't work out. The problem is only on Windows for Java 11.